gp_emulator/GaussianProcess.py

```diff
@@ -283,13 +368,14 @@ def hessian ( self, testing ):
     #target2 = data[ :,1]
     #target3 = data[ :,2]
     #input_obs = data[ :, 3: ]
-    wheat = np.loadtxt("argentine_wheat.dat")[:, 1:]
+    wheat = np.loadtxt("../data/argentine_wheat.dat")[:, 1:]
     yields = wheat[:,0]
     mu = yields.mean()
     sigma = yields.std()
     yields = (yields - mu ) / sigma
     wheat [:, 0] = yields
     rmse = []
+    print wheat.shape
```

Comment:
Avoid print statements for debugging.

Use the python `logging` library instead. With log.debug('foo').
Then you can turn your debug statements on and off by configuring your logger.